### PR TITLE
Correct log entries.

### PR DESCRIPTION
**Pull request by neunhoef**
This addresses BTS-114. This is a backport of the corresponding `devel`
change.

### CODE REVIEW

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10709/

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10713/

File: lib/Logger/LoggerFeature.h [hunk 1]
```diff
@@ -79,6 +82,8 @@ class LoggerFeature final : public application_features::ApplicationFeature {
   bool _supervisor = false;
   bool _backgrounded = false;
   bool _threaded = false;
+  std::string _apiSwitch;
+  bool _apiEnabled;
```
Comment (joerg84) [inline lib/Logger/LoggerFeature.h:86]: The other bool switches are explicitly set.

Comment (neunhoef) [reply lib/Logger/LoggerFeature.h:86]: True, I have used the old-fashioned way to initialize in the Constructor. Should I change this? I would have to do this in 5 PRs and run Jenkins 5 more times. I can do this, if we think it is worthwhile...

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10765/

**Comment by neunhoef** [comment]
Needs one more run.

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10799/

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10816/

**Comment by neunhoef** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10819/

**Comment by KVS85** [comment]
Tests blue.

**Pull request merged**